src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,58 @@
+package duke.parser;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+/**
+ * Represents a test class for Parser
+ */
+public class ParserTest {
+    private Parser parser;
+    private Ui ui;
+    private Storage storage;
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
```

Comment:
private final constants should be uppercase, with separating underscores

```suggestion
    private final ByteArrayOutputStream OUT_CONTENT = new ByteArrayOutputStream();
```